File: JavaScript/프로그래머스/0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    var answer = 0;
    for(var i=1; i<=n;i++){
        var count=0;
        for(var j=1;j<=i;j++){
            if(i%j==0) count++;
        }
        if(count>=3) answer++;
    }
    return answer;
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int count=0;
    int count2=0;
    for(int i=1; i<=n ;i++){
        count=0;
        for(int j=1; j<=i;j++){
            if(i%j==0) count++;
        }
        if(count>=3) count2++;
    }
    answer=count2;
    return answer;
}